tests/unit/transports/test_sms.py

```diff
@@ -0,0 +1,95 @@
+import uuid
+
+from flask import Response
+
+from funnel.transports.base import TransportRecipientError
+from funnel.transports.sms import (
+    TransportTransactionError,
+    make_exotel_token,
+    send,
+    validate_exotel_token,
+)
+
+# Target Numbers (Test Only). See this
+# https://www.twilio.com/docs/iam/test-credentials
+TWILIO_CLEAN_TARGET = "+15005550010"
+TWILIO_INVALID_TARGET = "+15005550001"
+TWILIO_CANT_ROUTE = "+15005550002"
+TWILIO_NO_SMS_SERVICE = "+15005550009"
+
+# Exotel Numbers to test. They are just made up numbers.
+EXOTEL_TO = "+919999999999"
```

Comment:
We can't do this unless Exotel gives us a test API. Instead, mock the request to fake the API call. Use [requests-mock](https://requests-mock.readthedocs.io/en/latest/pytest.html)

Reply:
Sample use of requests-mock is in `tests/unit/forms/test_project_forms.py`

Reply:
A monkey patch of the requests object via mock was sufficient as there was no need to test responses.